src/frontend/komponenter/ManuellJournalføring/ManuellJournalføring.tsx

```diff
@@ -53,163 +68,183 @@ const ManuellJournalføringContent: React.FC = () => {
+                        />
+                        {visModal && (
+                            <UIModalWrapper
+                                modal={{
+                                    className: 'søknad-modal',
+                                    tittel: 'Ønsker du å journalføre uten å knytte til behandling?',
+                                    lukkKnapp: false,
+                                    visModal: visModal,
+                                    actions: [
+                                        <Knapp
+                                            key={'ja'}
+                                            type={'hoved'}
+                                            mini={true}
+                                            spinner={senderInn}
+                                            disabled={senderInn}
+                                            onClick={() => {
+                                                settVisModal(false);
+                                                manueltJournalfør();
+                                            }}
+                                            children={'Ja, journalfør'}
+                                        />,
+                                        <Knapp
+                                            key={'nei'}
+                                            mini={true}
+                                            onClick={() => {
+                                                settVisModal(false);
+                                            }}
+                                            children={
+                                                behandlinger && behandlinger.length > 0
+                                                    ? hentAktivBehandlingForJournalføring()
+                                                        ? 'Nei, velg behandling'
+                                                        : 'Nei, velg/opprett behandling'
+                                                    : 'Nei, opprett behandling'
+                                            }
+                                        />,
+                                    ],
+                                }}
+                            >
+                                <Normaltekst className={'søknad-modal__fjern-vilkår-advarsel'}>
+                                    Du har valgt å journalføre uten å knytte dokumentet til en
+                                    spesifikk behandling. Journalposten knyttes kun til personen.
+                                    <br />
+                                    (Tilsvarende "Knytt til generell sak" i Gosys).
+                                </Normaltekst>
+                            </UIModalWrapper>
+                        )}
+                    </Skjemasteg>
+                    {visDokument && dokumentData.status === RessursStatus.SUKSESS && (
+                        <Bakgrunn>
```

Comment:
Could we just style the iframe directly?

Reply:
Oh yes, you are right and we fixed it!